### PR TITLE
Rails initial commit

### PR DESCRIPTION
**Pull request by DandyCV**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,67 +1,24 @@
...
-# if using bower-rails ignore default bower_components path bower.json files
-/vendor/assets/bower_components
-*.bowerrc
-bower.json
-
-# Ignore pow environment settings
-.powenv
-
-# Ignore Byebug command history file.
 .byebug_history
```
Comment (bestwebua) [inline .gitignore:22]: .DS_Store

File: .gitignore [hunk 2]
```diff
@@ -1,67 +1,24 @@
...
-rerun.txt
-pickle-email-*.html
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
+
+# Ignore bundler config.
+/.bundle
```
Comment (bestwebua) [inline .gitignore:8]: /vendor/bundle

File: .ruby-version [hunk 3]
```diff
@@ -0,0 +1 @@
+3.0.1
```
Comment (bestwebua) [inline .ruby-version:1]: .ruby-version or .tool-versions, 2 specefiers are unnecessary.

**Review by bestwebua — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (DandyCV) [reply .gitignore:22]: added

File: .gitignore (hunk 2 above)
Comment (DandyCV) [reply .gitignore:8]: added

File: .ruby-version (hunk 3 above)
Comment (DandyCV) [reply .ruby-version:1]: .tool-versions deleted

**Review by bestwebua — APPROVED** [review]
👍

**Pull request merged**